awscli/examples/ec2/describe-vpc-classic-link-dns-support.rst

```diff
@@ -0,0 +1,22 @@
+**To describe ClassicLink DNS support for your VPCs**
+
+This example describes the ClassicLink DNS support status of all of your VPCs. 
+
+Command::
+
+  aws ec2 describe-vpc-classic-link-dns-support
+
+Output::
+
+  {
+    "Vpcs": [
+      {
+        "VpcId": "vpc-614cc409", 
```

Comment:
I think the difference is that that the `vpc-888888` is for inputs and this is showing output.  Looking at other examples, this seems to be consistent, so I think this change is fine as is.